docs/src/content/docs/fr/guides/components.mdx

```diff
@@ -109,3 +123,28 @@ Ajoutez l'attribut `stagger` pour décaler verticalement la deuxième colonne de
 ```
 
 :::
+
+### Icon
```

Comment:
In the current translation, `Tabs` is translated to `Onglets`, `Cards` is not translated. It looks like most translations translate the component names so I guess we should do the same. If so, that would mean translating `Cards` too I guess. Any thoughts?

```suggestion
### Icônes
```